source/dotnet/Library/AdaptiveCards.Rendering.Wpf.Xceed/XceedDateInput.cs

```diff
@@ -40,6 +40,12 @@ public static FrameworkElement Render(AdaptiveDateInput input, AdaptiveRenderCon
                 datePicker.Style = context.GetStyle("Adaptive.Input.Date");
                 datePicker.DataContext = input;
 
+                if (input.IsRequired && input.ErrorMessage == null)
```

Comment:
What;s our behavior for empty or whitespace-only values for the ErrorMessage?

Reply:
Good point. Fixed.

---
In reply to: [501254812](https://github.com/microsoft/AdaptiveCards/pull/4885#discussion_r501254812) [](ancestors = 501254812)